src/NpmUtilities.js

```diff
@@ -33,6 +33,10 @@ export default class NpmUtilities {
     const packageJson = path.join(directory, "package.json");
     const packageJsonBkp = packageJson + ".lerna_backup";
 
+    // Grab the original package.json so that we can inject a few properties to
+    // the temporary fake replacement.
+    const originalPackage = JSON.parse(FileSystemUtilities.readFileSync(packageJson));
```

Comment:
I initially had a regular `require(packageJson)` instead of reading the file through FileSystemUtilites. But that was hard to test, plus I wasn't fully sure what the side-effects would be (in terms of caching etc).